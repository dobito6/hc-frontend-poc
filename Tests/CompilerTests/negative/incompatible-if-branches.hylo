public fun main() {
  _ = if true { (a: ()) } else { (b: ()) }
}
