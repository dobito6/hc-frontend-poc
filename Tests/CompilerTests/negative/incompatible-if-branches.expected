negative/incompatible-if-branches.hylo:2.34-41: error: no coercion from '{b: Void}' to '{a: Void}'
  _ = if true { (a: ()) } else { (b: ()) }
                                 ~~~~~~~
